tests/ha/single_pod_restart/test_server_pod_restart_api.py

```diff
@@ -1130,5 +1259,192 @@ def test_obj_ver_suspension_server_pod_restart(self):
         """
         LOGGER.info("STARTED: Test to verify bucket versioning suspension before & after server "
                     "pod restart.")
+        event = threading.Event()
+        LOGGER.info("Step 1: Create bucket and upload object %s of %s size. Enable versioning "
+                    "on %s.", self.object_name, self.f_size, self.bucket_name)
+        self.extra_files.append(self.multipart_obj_path)
+        args = {'chk_null_version': True, 'is_unversioned': True, 'file_path':
+            self.multipart_obj_path, 'enable_ver': True, 's3_ver': self.s3_ver}
+        resp = self.ha_obj.create_bkt_put_object(event, self.s3_test_obj, self.bucket_name,
```

Comment:
forgot this in last PR but maybe we need some more relevant lib name as its enabling versioning as well.
like create_bkt_put_obj_ver